build/tasks/AddArchiveReferences.cs

```diff
@@ -52,12 +52,28 @@ public override bool Execute()
             foreach (var package in archiveArtifacts)
             {
                 var packageName = package.ItemSpec;
-                var packageVersion = string.Equals(packageName, "Microsoft.AspNetCore.All", StringComparison.OrdinalIgnoreCase) ?
-                    MetaPackageVersion :
-                    buildArtifacts
-                        .Single(p => string.Equals(p.PackageInfo.Id, packageName, StringComparison.OrdinalIgnoreCase))
-                        .PackageInfo.Version.ToString();
 
+                string packageVersion;
+
+                if (string.Equals(packageName, "Microsoft.AspNetCore.All", StringComparison.OrdinalIgnoreCase)
```

Comment:
Are there any other areas in the code that have hard-coded this package name? I'm concerned now that we're probably missing other places where we need to add .App too.

Reply:
I did a search in the Universe repo and i found another in lineups. Not sure what else might be affected. I'm not sure if there are any other repos affected. I also took a look at Coherence-Signed but didn't find anything.